integration-tests/tests/models/single.spec.ts

```diff
@@ -68,6 +68,49 @@ describe('publish', () => {
       expect: 'ignored',
     });
   });
+
+  test.concurrent('CLI output', async () => {
```

Comment:
```suggestion
  test.concurrent('CLI output', async ({ expect }) => {
```

I guess when using `test.concurrent` you need to destructure `expect`

Reply:
none of our `test.concurrent` tests gets `expect` like this

Reply:
oops, you need to destructure only if you use `toMatchInlineSnapshot` inside `test.concurrent` in vitest

Reply:
:)